packages/generated-project/overrides/app/pages/cart/index.jsx

```diff
@@ -0,0 +1,95 @@
+import CartTitle from '@salesforce/retail-react-app/app/pages/cart/partials/cart-title'
+import EmptyCart from '@salesforce/retail-react-app/app/pages/cart/partials/empty-cart'
+import OrderSummary from '@salesforce/retail-react-app/app/components/order-summary'
+
+import {useCurrentBasket} from '@salesforce/retail-react-app/app/hooks/use-current-basket'
+import {useCurrentCustomer} from '@salesforce/retail-react-app/app/hooks/use-current-customer'
+import {
+    HideOnDesktop,
+    HideOnMobile
+} from '@salesforce/retail-react-app/app/components/responsive/index'
+import ProductRecommendations from '@salesforce/retail-react-app/app/pages/cart/product-recommendations'
+import {SelectedItemContext} from '@salesforce/retail-react-app/app/pages/cart/use-selected-item'
+import ProductItems from '@salesforce/retail-react-app/app/pages/cart/product-items'
+
+const Cart = () => {
+    const [selectedItem, setSelectedItem] = useState(undefined)
+    const {data: basket, isLoading} = useCurrentBasket()
+    const {data: customer} = useCurrentCustomer()
+    const {isRegistered} = customer
+
+    if (isLoading) {
+        return <CartSkeleton />
+    }
+
+    if (!isLoading && !basket?.productItems?.length) {
+        return <EmptyCart isRegistered={isRegistered} />
+    }
+    return (
+        <SelectedItemContext.Provider value={{selectedItem, setSelectedItem}}>
+            <Box background="gray.50" flex="1" data-testid="sf-cart-container">
+                <Container
+                    maxWidth="container.xl"
+                    px={[4, 6, 6, 4]}
+                    paddingTop={{base: 8, lg: 8}}
+                    paddingBottom={{base: 8, lg: 14}}
+                >
+                    <Stack spacing={24}>
+                        <Stack spacing={4}>
+                            <CartTitle />
+
+                            <Grid
+                                templateColumns={{base: '1fr', lg: '66% 1fr'}}
+                                gap={{base: 10, xl: 20}}
+                            >
+                                {/* NOTE: rearranging the content so that the order summary comes first */}
+                                <GridItem>
+                                    <Stack spacing={4}>
+                                        <OrderSummary
+                                            showPromoCodeForm={true}
+                                            isEstimate={true}
+                                            basket={basket}
+                                        />
+                                        <HideOnMobile>
+                                            <CartCta />
+                                        </HideOnMobile>
+                                    </Stack>
+                                </GridItem>
+                                <GridItem>
+                                    <ProductItems />
+                                </GridItem>
+                            </Grid>
```

Comment:
This file shows an example of overriding to _replace_ (rather than extend). You may have to resort to doing that for complex customizations. 

Here I simply reorder the content around.